lib/services/database.go

```diff
@@ -686,20 +675,44 @@ func NewDatabaseFromRDSV2Cluster(cluster *rdsTypesV2.DBCluster) (types.Database,
 		})
 }
 
+func rdsSubnetGroupToNetworkInfo(subnetGroup *rdsTypesV2.DBSubnetGroup) (vpcID string, subnets []string) {
+	if subnetGroup != nil {
+		vpcID = aws.StringValue(subnetGroup.VpcId)
+		subnets = make([]string, 0, len(subnetGroup.Subnets))
+		for _, s := range subnetGroup.Subnets {
+			subnetID := aws.StringValue(s.SubnetIdentifier)
+			if subnetID != "" {
+				subnets = append(subnets, subnetID)
+			}
+		}
+	}
+	return
+}
+
 // MetadataFromRDSV2Cluster creates AWS metadata from the provided RDS cluster.
 // It uses aws sdk v2.
-func MetadataFromRDSV2Cluster(rdsCluster *rdsTypesV2.DBCluster) (*types.AWS, error) {
+func MetadataFromRDSV2Cluster(rdsCluster *rdsTypesV2.DBCluster, rdsInstance *rdsTypesV2.DBInstance) (*types.AWS, error) {
 	parsedARN, err := arn.Parse(aws.StringValue(rdsCluster.DBClusterArn))
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
+
+	var vpcID string
+	var subnets []string
+
```

Comment:
nit: Keep declaration and assignment in the same block?

```suggestion
```

Reply:
Not sure I follow.

The assignment is conditional, based on the presence of `rdsInstance`.

Reply:
There isn't a lot to follow, feel free to push back.